CMakeLists.txt

```diff
@@ -60,6 +60,7 @@ option(USE_IMAGE_API  "Build tiny-dnn with Image API support"       ON)
 option(USE_GEMMLOWP   "Build tiny-dnn with gemmlowp support"       OFF)
 
 option(BUILD_TESTS      "Set to ON to build tests"              OFF)
+option(USE_SYS_GTEST  "Set to ON to use a system installation of gtest" OFF)
```

Comment:
@morxa I like when people ignore the beauty of indentation. That's how they optimize their precious time.

Reply:
As I said in the original PR, I can rebase onto master if necessary.

Reply:
It would be great if you would do that. I see `CMakeLists.txt` is conflicting, not sure why though.

Unfortunately (or fortunately?), there're many tests required to pass before merging pull requests. So it can be a daunting process. Please rely on your coffee anytime.